src/config/env/heroku.js

```diff
@@ -0,0 +1,12 @@
+export default {
+    env: 'heroku',
+    db: 'mongodb://garage-it:smarthause@ds017553.mlab.com:17553/garage-it',
+    port: process.env.PORT,
+    seedDB: false,
+    mqtt: {
+        port: 12787,
+        hostname: 'm21.cloudmqtt.com',
```

Comment:
!fix
I guess, In this case I will hide this in ```process.env```
What your opinion or suggestion about this?